pkg/system/network/linux.go

```diff
@@ -198,3 +198,14 @@ func (t NetworkManagerLinux) TryConnect(nixPatch dogeboxd.NixPatch) error {
 	log.Printf("Successfully saved network configuration to disk")
 	return nil
 }
+
+func (t NetworkManagerLinux) GetLocalIP() (net.IP, error) {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
```

Comment:
This bakes in google DNS ?

Reply:
Yeah, happy to change to whatever we want, `dogecoin.org` maybe? Just needs to be an external-to-the-box address so that we can solve which IP is used for outbound traffic.